### ISSUE TITLE
Set default compression type

### ISSUE THREAD

**Issue by anhnmt** [redpanda-data/console#1405]
![image](https://github.com/user-attachments/assets/01e12c56-7516-4663-a2ef-cd3f90efa178)

Is there any way I can leave the default Compress type as UNCOMPRESSED?
I'm testing old programs and having to select UNCOMPRESSED every time is inconvenient

**Comment by weeco** [comment]
We could store the last selection in the frontend. That would be relatively easy to add, do you want to give it a shot?

**Comment by anhnmt** [comment]
@weeco Of course, I really need this feature. Also, if you can do a message recall feature, it would be even better.

![image](https://github.com/user-attachments/assets/6966f7de-ff7c-4a9f-b1fc-169ec56c4038)

Sometimes I want to reuse old messages, this will help save more time

**Comment by weeco** [comment]
What do you mean by a recall feature? Reproducing an existing message and possibly editing it along the way?

**Comment by anhnmt** [comment]
@weeco Sorry I don't know what to call this function, I just want to copy the old message and push it to the end of the topic with all the information from before.

**Comment by real-mday** [comment]
This issue is currently causing us to get spikes of >1 million messages over the course of hours on our development cluster when a test message is created and our software currently is not able to recover from trying to read a compressed message
